engines/scumm/he/logic/football.cpp

```diff
@@ -691,6 +695,11 @@ int LogicHEfootball2002::netInitLanGame(int32 *args) {
 		// Stop querying sessions if we haven't already
 		_vm->_net->stopQuerySessions();
 		// And host our new game.
+		// If there's a custom game name, use that instead.
+		if (ConfMan.hasKey("game_session_name")) {
+			Common::String gameSessionName = ConfMan.get("game_session_name");
+			return _vm->_net->hostGame(const_cast<char *>(gameSessionName.c_str()), userName);
```

Comment:
`c_str()` is already `char *`. Is this cast really needed?

Reply:
`c_str()` actually returns an `const char*`, which will result in an invalid conversion error.  So this cast is indeed needed.